volatility3/framework/plugins/linux/pagecache.py

```diff
@@ -520,8 +521,13 @@ def _generator(self):
 
         if self.config["dump"]:
             filename = self.config["dump"]
```

Comment:
This line can be removed since the `dump` option is now a boolean.

Reply:
thanks.. good catch! I just removed it. It should be all set once the test cases complete successfully.